__tests__/app.test.js

```diff
@@ -0,0 +1,28 @@
+const request = require("supertest");
+const app = require("../db/app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+
+beforeEach(() => {
+  return seed(db);
+});
+afterAll(() => db.end());
+
+describe("GET topics", () => {
+  test("should respond with a 200 status code", () => {
+    return request(app).get("/api/topics").expect(200);
+  });
+  test("should return an array of all topic objects with correct keys", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((res) => {
+        console.log(res.body.topics.rows);
+        expect(Array.isArray(res.body.topics.rows)).toBe(true);
+        res.body.topics.rows.forEach((topics) => {
+          expect(typeof topics.slug).toBe("string");
+          expect(typeof topics.description).toBe("string");
```

Comment:
This is a good test 👍  I'd suggest having the data on a key of just `topics` , having `rows` as well isn't necessary